Author: Steve Kemp <skx@debian.org>
Description: Missing openssl  includes in sshcrypto.c. 
This patch was through diff.gz and now is implemented as a dpatch.
--- a/sshcrypto.c	2011-06-19 17:13:57.971999389 -0500
+++ b/sshcrypto.c	2011-06-19 17:15:19.847999373 -0500
@@ -14,6 +14,8 @@
 
 #include <sys/types.h>
 #include <openssl/ssl.h>
+#include <openssl/blowfish.h>
+#include <openssl/des.h>
 
 #include <err.h>
 #include <stdio.h>
